### PR TITLE
e2e:update e2e test script and quick start script

### PR DESCRIPTION
**Pull request by village-way**
#### What type of PR is this?
/kind failing-test

#### What does this PR do?

update e2e script

#### Which issue(s) does this PR fix?

Fixes #
#### Special notes for your reviewer:

@duanmengkk 
#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

File: hack/util.sh [hunk 1]
```diff
@@ -542,4 +542,56 @@ function util::go_clean_cache() {
...
+
+# util::get_macos_ipaddress will get ip address on macos interactively, store to 'MAC_NIC_IPADDRESS' if available
+MAC_NIC_IPADDRESS=''
+function util::get_macos_ipaddress() {
+  if [[ $(go env GOOS) = "darwin" ]]; then
+    tmp_ip=$(ipconfig getifaddr en0 || true)
+    echo ""
+    echo " Detected that you are installing Karmada on macOS "
+    echo ""
+    echo "It needs a Macintosh IP address to bind Karmada API Server(port 5443),"
```
Comment (duanmengkk) [inline hack/util.sh]:
```suggestion
    echo "It needs a Macintosh IP address to bind Kosmos API Server(port 5443),"
```

File: hack/util.sh [hunk 2]
```diff
@@ -542,4 +542,56 @@ function util::go_clean_cache() {
     go clean -cache
 
     set +x
+}
+
+# 函数定义，根据操作系统类型获取kubeconfig的base64编码
```
Comment (duanmengkk) [inline hack/util.sh]: remove the  comment or use english

Comment (village-way) [reply hack/util.sh]: fixed

File: hack/util.sh (hunk 1 above)
Comment (village-way) [reply hack/util.sh]: fixed

**Pull request merged**